vuepress/docs/next/docs/consume/entando-apis.md

```diff
@@ -3,139 +3,80 @@ sidebarDepth: 2
 
-### API structure
+### Disable Swagger UI
 
-All of the APIs share a common top level structure. Each response will
-contain a top level entry for `errors`, `metadata`, and `payload`.
+Repeat the steps above, but in step 4, remove `swagger` from the value list.
 
-The `errors` will always contain code and a message string indicating an
-error condition in the request. The `metadata` section is used for
-paging, sorting, filtering and data that is distinct from the body. The
-body of each response is included in the `payload` section of the
-response and varies according to each API.
+## Find Your Client Secret
+You'll need your client credentials to execute the Entando APIs. 
 
-### Models
+1. Login into your Keycloak Administration Console. To find the Kubernetes credentials, see the [Entando Identity Management System](./identity-management.md) page.
 
-All of the model classes returned by the Entando core are annotated so that the model definition is included in the Swagger documentation. At the bottom of the Swagger page all of the model classes returned by the API endpoints can be found.
+2. From the left nav, go to `Clients`
 
-## Tutorial
+3. Select the desired client (e.g. in a quickstart environment this is `quickstart`)
 
-1. Access your application Swagger UI as discussed above
+4. Click on the `Credentials` tab to retrieve the secret 
 
-2. Click on the `Authorize` button in the upper right corner
+## Access the APIs on Swagger
+1.  To see the APIs, go to: 
+``` sh
+http://[YOUR-HOST-NAME]/entando-de-app/api/swagger-ui.html
+```
 
-3. Enter the client id and client secret in the open window and click `Authorize`
+2. Click on the Authorize button in the upper right corner.
 
-4. If you are redirected to the Entando login page, log in with your credentials (default are `admin`/`adminadmin`)
+3. Enter the client id and secret and click Authorize.
 
-5. You will be redirected to the Swagger UI page, now authenticated
+4. You will be prompted to log in to your Keycloak instance as an admin user if you have not already done so. The defaults are admin/adminadmin. 
 
-6. Use the **Try it out** button on the APIs
+5. You will be redirected to the authenticated Swagger UI page. Select an API to see the methods in the drop-down list and click the `Try it out` button.  
+Example:
+   1. Scroll down to `widget-controller`
 
-    -   Scroll to `widget-controller`
+   2. Select the `GET` method
 
-    -   Select the blue GET row
+   3. Click `Try it out`
 
-    -   Select **Try it out**
+   4. See the results in the window below. You should see a Server response with Code 200 and full response body.
 
-    -   Look at the results in the window. You should see a Server response with Code 200 and full response body.
\ No newline at end of file
+### Next Steps
+* For information, see the tutorial on [adding a REST API](../../tutorials/devops/add-rest-api.md).
+* Learn how to [invoke Entando core APIs](../../tutorials/devops/invoking-api.md).
```

Comment:
here is Entando core again! are "entando" and "core" modifiers for "APIs" or is "Entando" modifying "core APIs"? like are we talking about the Entando core or core APIs? if the entando core is a thing we're referencing often we need to better define it in an intro page

Reply:
no mention of "core" here: https://developer.entando.com/next/docs/getting-started/concepts-overview.html#key-entando-concepts

Reply:
Not touching this one because it is the tutorial name and I think it refers to core in the sense that it's attached to the center of Entando which is the Engine. If we want to change that title, we can get a ticket for that. Core and engine are used interchangeably by the devs apparently

Reply:
got rid of both next steps because the REST apis is going to be deleted anyways

Reply:
or we could make a ticket for defining core somewhere in the docs... it appears here and there but is never defined